Kit 527 pipeline for building and testing solana packages

dantedelucia opened the pull request:
Required changes to allow packages overrides.

alexanderguy commented on index.ts:
```diff
@@ -45,7 +45,14 @@ const clstr = new spe.Cluster(
+clstr.addAgaveMember(node0, {
+  runnerConfig: {
+    packageConfig: {
+      overrideDir: "build",
```
Please use pulumi config for this, rather than making it static.

alexanderguy committed:
prettier

alexanderguy committed:
Update to pulumi-svmkit v0.51.0

alexanderguy committed:
Support providing overrideDir via pulumi config

This allows you to have a central spot to dump packages, which
will be used by the SVMKit components instead of the software
stored inside apt.

alexanderguy committed:
Change project name and description
